src/modules/drawing/DrawingModule.vue

```diff
@@ -293,22 +296,44 @@ export default {
             clearTimeout(this.KMLUpdateTimeout)
             const kml = generateKmlString(this.drawingLayer.getSource().getFeatures())
             if (kml && kml.length) {
+                this.savingStatus = 'draw_file_saving'
                 try {
                     await this.saveDrawing(kml)
                 } catch (e) {
                     log.error('Could not save KML layer: ', e)
+                    // We should probably add a translation key for that
+                    this.savingStatus = 'Error while saving the drawing'
                     throw e
                 }
+                if (this.savingStatus !== '') {
+                    this.savingStatus = 'draw_file_saved'
+                }
             }
         },
         onChange() {
+            this.willModify()
             this.$nextTick(() => {
                 this.isDrawingEmpty = this.drawingLayer.getSource().getFeatures().length === 0
                 this.triggerKMLUpdate()
             })
         },
         onDrawStart(feature) {
             this.currentlySketchedFeature = feature
+            /* Do not call willModify() here as we are not sure if there will be any modification
+            (maybe the user decides to cancel the drawing) */
+        },
+
+        /**
+         * Call this method when there are or when there will be unsaved changes. Change the saving
+         * status to "possibly unsaved changes"
+         */
+        willModify() {
+            if (
+                this.savingStatus === 'draw_file_saved' ||
+                this.savingStatus === 'draw_file_saving'
+            ) {
+                this.savingStatus = ''
+            }
```

Comment:
Would it not make more sense to have something like this
```javascript
clearStatus() {
  if (this.savingStatus !== '') {
    this.savingStatus = ''
  }
}
```
Because with you code above if the status is an error then it will not be erased.

Reply:
The idea of this if statement was to make the error message a bit more persistant / to not discard the error message before the next saving attempt is made. So the error message can be replaced the the draw_file_saved or draw_file_saving status, but not by the empty status.

Reply:
I see make sense. I'm not a big fan of such string compare which to make it worse are translation dependent. 
It might wise to either use a `savingStatus` as an enum and in the rendering translate the enum to the corresponding string internationalized. This way the `DrawingModule` is not responsible to choose what will be presented to the user, it is only responsible to notify the status as an enum, then `DrawingToolbox` would be responsible for the rendering choosing the correct way to render (using translation or any fancy stuff). `DrawingToolbox` is the component that will render the status, then it should have fully control on what it will display based on a status flag.